include/core/nexus/HY_PointHydroNexusRemote.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include <unordered_map>
 #include <string>
+#include <list>
```

Comment:
I agree this was missing, but how did it get discovered in this PR?

Reply:
Building with `NGEN_WITH_MPI=ON` provided that -- not sure how it was missed previously